ui.go

```diff
@@ -52,6 +52,11 @@ func (u *UI) Update() {
 
 	u.handleFocusChangeRequest()
 
+	//If widget is not visible or disabled, change focus to next widget
```

Comment:
I think there is another change we need here. We need to update Container and Scroll container GetFocusers method to add a IsVisible check

```golang
		case Focuser:
			if widget, ok := v.(HasWidget); ok {
				if v.TabOrder() >= 0 && !widget.GetWidget().Disabled {
					result = append(result, v)
				}
			}
```

Reply:
Okay! I will update it, I thought about it when I did this but when I tried it it behaved as supposed, if the next element was not visible it jumped over to the next one, but just in case it's better to have that check there as well